### ISSUE TITLE
Unhandled Exception running cfn init(rpdk.core.project package was not installed in a way that PackageLoader understands)

### ISSUE THREAD

**Issue by jayzandme** [aws-cloudformation/cloudformation-cli#758]
Followed the steps in the README(https://github.com/aws-cloudformation/cloudformation-cli)

Getting an Unhandled Exception when running cfn init

=== Unhandled exception ===
Please report this issue to the team.
Issue tracker: github.com/aws-cloudformation/cloudformation-cli/issues
Please include the log file 'rpdk.log'

[rpdk.log](https://github.com/aws-cloudformation/cloudformation-cli/files/6474628/rpdk.log)

**Comment by PatMyron** [comment]
similar to https://github.com/aws-cloudformation/cloudformation-cli/issues/757

**Comment by KMK-Git** [comment]
@PatMyron I faced the same error for `cfn init`. I resolved it by using Jinja2==2.11.3 instead of the latest Jinja2==3.0.0

`pip freeze` after `pip install cloudformation-cli cloudformation-cli-python-plugin`. `cfn init` fails with same error as above
```
attrs==21.2.0
aws-sam-translator==1.35.0
boto3==1.17.72
botocore==1.20.72
certifi==2020.12.5
cfn-flip==1.2.3
cfn-lint==0.49.1
chardet==4.0.0
click==8.0.0
cloudformation-cli==0.2.9
cloudformation-cli-python-plugin==2.1.2
colorama==0.4.4
decorator==4.4.2
docker==4.4.4
hypothesis==6.12.0
idna==2.10
importlib-metadata==4.0.1
iniconfig==1.1.1
Jinja2==3.0.0
jmespath==0.10.0
jsonpatch==1.32
jsonpointer==2.1
jsonschema==3.2.0
junit-xml==1.9
MarkupSafe==2.0.0
nested-lookup==0.2.22
networkx==2.5.1
ordered-set==4.0.2
packaging==20.9
pluggy==0.13.1
py==1.10.0
pyparsing==2.4.7
pyrsistent==0.17.3
pytest==6.2.4
pytest-localserver==0.5.0
pytest-random-order==1.0.4
python-dateutil==2.8.1
PyYAML==5.4.1
requests==2.25.1
s3transfer==0.4.2
six==1.16.0
sortedcontainers==2.3.0
toml==0.10.2
typing-extensions==3.10.0.0
urllib3==1.25.11
websocket-client==0.59.0
Werkzeug==2.0.0
zipp==3.4.1
```

`pip freeze` after `pip install Jinja2==2.11.3` and `pip install cloudformation-cli cloudformation-cli-python-plugin`. `cfn init` works correctly
```
attrs==21.2.0
aws-sam-translator==1.35.0
boto3==1.17.72
botocore==1.20.72
certifi==2020.12.5
cfn-flip==1.2.3
cfn-lint==0.49.1
chardet==4.0.0
click==8.0.0
cloudformation-cli==0.2.9
cloudformation-cli-python-plugin==2.1.2
colorama==0.4.4
decorator==4.4.2
docker==4.4.4
hypothesis==6.12.0
idna==2.10
importlib-metadata==4.0.1
iniconfig==1.1.1
Jinja2==2.11.3
jmespath==0.10.0
jsonpatch==1.32
jsonpointer==2.1
jsonschema==3.2.0
junit-xml==1.9
MarkupSafe==2.0.0
nested-lookup==0.2.22
networkx==2.5.1
ordered-set==4.0.2
packaging==20.9
pluggy==0.13.1
py==1.10.0
pyparsing==2.4.7
pyrsistent==0.17.3
pytest==6.2.4
pytest-localserver==0.5.0
pytest-random-order==1.0.4
python-dateutil==2.8.1
PyYAML==5.4.1
requests==2.25.1
s3transfer==0.4.2
six==1.16.0
sortedcontainers==2.3.0
toml==0.10.2
typing-extensions==3.10.0.0
urllib3==1.25.11
websocket-client==0.59.0
Werkzeug==2.0.0
zipp==3.4.1
```

**Comment by jayzandme** [comment]
Thank you! Yes downgrading jinja2 seems to have resolved the issue. Much appreciated!
